src/codemodder/codemods/test/integration_utils.py

```diff
@@ -288,21 +292,25 @@ def check_sonar_issues(cls):
             (cls.sonar_issues_json, cls.sonar_hotspots_json)
         )
 
-        assert (
-            cls.codemod.requested_rules[-1] in sonar_results
+        assert any(
+            map(lambda x: x in sonar_results, cls.codemod.requested_rules)
```

Comment:
list comp style would be preferable here
`any(x in sonar_results for x in cls.codemod.requested_rules)`